provision-tahoe.py

```diff
@@ -45,11 +50,24 @@ def move_environment_files_to_host():
             symlink(src_path, container_path)
 
 
-def main():
-    if not ENVS_DIR.exists():
+def install_auto_pip_requirements():
+    """
+    Install source pip packages (git repositories) that are checked out at `src/edxapp-pip`.
+
+    This useful to avoid the need to re-install pip requirements every time a `$ make dev.up` is done.
+    """
+    if not PIP_DIR.exists():
         makedirs(ENVS_DIR)
```

Comment:
should that be `PIP_DIR`?

I'm not sure why we'd need to create it though. I'd think that if the `PIP_DIR` doesn't exist this function could just return since there's clearly no package dirs in it to install.